src/System.IO.Packaging/src/System/IO/Packaging/SRID.cs

```diff
@@ -0,0 +1,305 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace System.IO.Packaging
+{
+    internal class SRID
```

Comment:
Should this library use the same resources design used by the rest of the corefx libraries?